src/main/resources/reference.conf

```diff
@@ -201,6 +206,7 @@ metadata-overrides {
               }
               balance {
                 visible: true
+                scale: 6
```

Comment:
This just means that balance is microtez, and to get the actual tez, we multiply by 10^6, correct?

Reply:
https://github.com/Cryptonomic/Conseil/issues/354#issuecomment-494889684

I think it's explained best there

Reply:
Thanks!